### ISSUE TITLE
How come different performance?

### ISSUE THREAD

**Issue by tessavdheiden** [MorvanZhou/pytorch-A3C#12]
Hi Morvan,

Why do I get totally different performance (see file attached)?

Best,
Tessa
![a3c_continious](https://user-images.githubusercontent.com/24938569/73439252-a0f35380-434f-11ea-9c97-09761df3ea3a.png)

**Comment by LoveRL** [comment]
Hi, 

How about changing the 'UPDATE_GLOBAL_ITER' more than 5?

It was helpful to me.

**Comment by pengzhi1998** [comment]
Hi, but when I ran this code, the moving average reward is always below -1000 for the continuous situation, do you know what kind of problem it could be? (the 'UPDATE_GLOBAL_ITER' has already been set to 10) The performance of the discrete situation is very bad as well.

**Comment by LoveRL** [comment]
Hi, 

Here is another trial.

Try 'torch.nn.utils.clip_grad_norm_(lnet.parameters(), 20)' in utils.py 

![image](https://user-images.githubusercontent.com/24192196/76677970-32283c00-6617-11ea-91d4-2725de20869d.png)

**Comment by pengzhi1998** [comment]
> Hi,
> 
> Here is another trial.
> 
> Try 'torch.nn.utils.clip_grad_norm_(lnet.parameters(), 20)' in utils.py
> 
> ![image](https://user-images.githubusercontent.com/24192196/76677970-32283c00-6617-11ea-91d4-2725de20869d.png)
> 
> It helped me to reduce performance differences.

Thank you! I'll take a try.

**Comment by Eify666666** [comment]
> Hi,
> 
> Here is another trial.
> 
> Try 'torch.nn.utils.clip_grad_norm_(lnet.parameters(), 20)' in utils.py
> 
> ![image](https://user-images.githubusercontent.com/24192196/76677970-32283c00-6617-11ea-91d4-2725de20869d.png)
> 
> It helped me to reduce performance differences.

Hi，I meet a trouble when  I train another A3C.
After some time, all the networks always output the same action.
I tried the "torch.nn.utils.clip_grad_norm_(lnet.parameters(), 20)", it doesn't work.
It may be that during the training process, the network tries many times, but does not reap the reward.
Do you have any ideas about this problem?
